src/main/java/frc/robot/subsystems/gripper/GripperLoggedInputs.java

```diff
@@ -0,0 +1,28 @@
+package frc.robot.subsystems.gripper;
+
+import org.littletonrobotics.junction.LogTable;
+import org.littletonrobotics.junction.inputs.LoggableInputs;
+
+public class GripperLoggedInputs implements LoggableInputs {
+    boolean leftSolenoid;
+    boolean rightSolenoid;
```

Comment:
Maybe you should give more descriptive names for these fields, like - `leftSolenoidExtended`.